### PR TITLE
docs: use embed

### PR DESCRIPTION
**Pull request by crazyair**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [ ] Doc is updated/provided or not needed
- [ ] Demo is updated/provided or not needed
- [ ] TypeScript definition is updated/provided or not needed
- [ ] Changelog is provided or not needed

---

### <samp>🤖 Generated by Copilot at 166aa1d</samp>

Updated the documentation of `Space` and `Flex` components in Chinese to improve consistency and avoid duplication. Used an embed tag to reference the comparison section from `components/flex/index.zh-CN.md` in `components/space/index.zh-CN.md`.

### <samp>🤖 Generated by Copilot at 166aa1d</samp>

*  Update the comparison between Space and Flex components in Chinese documentation ([link](https://github.com/ant-design/ant-design/pull/45280/files?diff=unified&w=0#diff-1f6b9ec85e9c10788dfb367d525109c02a6430b23bdd94bf2617c5e92f2220e4L18-R18), [link](https://github.com/ant-design/ant-design/pull/45280/files?diff=unified&w=0#diff-3386c4be6e559b15e5fb0ee60814a492c147f3527a4eea4beebdf1855dd0c368L20-R21))
   - Rename the subsection title in `components/flex/index.zh-CN.md` to include both component names ([link](https://github.com/ant-design/ant-design/pull/45280/files?diff=unified&w=0#diff-1f6b9ec85e9c10788dfb367d525109c02a6430b23bdd94bf2617c5e92f2220e4L18-R18))
   - Replace the duplicated content in `components/space/index.zh-CN.md` with an embed tag that references the same subsection in `components/flex/index.zh-CN.md` ([link](https://github.com/ant-design/ant-design/pull/45280/files?diff=unified&w=0#diff-3386c4be6e559b15e5fb0ee60814a492c147f3527a4eea4beebdf1855dd0c368L20-R21))

### CODE REVIEW

File: components/space/index.zh-CN.md [hunk 1]
```diff
@@ -17,10 +17,7 @@ coverDark: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*37T2R6O9oi0AAA
 - 可以设置各种水平对齐方式。
 - 需要表单组件之间紧凑连接且合并边框时，使用 Space.Compact（自 `antd@4.24.0` 版本开始提供该组件）。
 
-### 与 Flex 组件的区别
-
-- Space 为内联元素提供间距，其本身会为每一个子元素添加包裹元素用于内联对齐。适用于行、列中多个子元素的等距排列。
-- Flex 为块级元素提供间距，其本身不会添加包裹元素。适用于垂直或水平方向上的子元素布局，并提供了更多的灵活性和控制能力。
+<embed src="../flex/index.zh-CN.md#Space 与 Flex 组件的区别"></embed>
```
Comment (PeachScript) [inline components/space/index.zh-CN.md:20]: 目前没有标题内容引用的功能，要用正则匹配获取内容，可以参考 dumi 官网的：https://github.com/umijs/dumi/blob/6590fa9ae3c54a28ea4e5fbcfdef0af1a643eb90/docs/config/markdown.md#L176

---

标题引用的能力可以加一下，细节没想好，按语雀的逻辑引用标题会包含标题自身，在这个场景下也不适用了

Comment (crazyair) [reply components/space/index.zh-CN.md:20]: 要不是为了能搜索，直接改用自定义组件了

Comment (crazyair) [reply components/space/index.zh-CN.md:20]: @PeachScript 是否可以这样
`<embed id='1'></embed>`

```
<embed-source id='1'>
### 与 Flex 组件的区别

- xxxxx
</embed-source>
```

Comment (PeachScript) [reply components/space/index.zh-CN.md:20]: 这会影响 Markdown 预览

Comment (crazyair) [reply components/space/index.zh-CN.md:20]:
```
<embed-source id='1'></embed-source>
### 与 Flex 组件的区别

- xxxxx
<embed-source id='1'></embed-source>
```

Comment (crazyair) [reply components/space/index.zh-CN.md:20]: 加个空行就没有影响

image

image

**Pull request closed without merging**